drivers/vulkan/vulkan_context.cpp

```diff
@@ -2473,7 +2477,7 @@ Error VulkanContext::swap_buffers() {
 		return OK;
 	}
 
-	//	print_line("swapbuffers?");
+	//	print_line("swap_buffers");
```

Comment:
```suggestion
	//print_line("swap_buffers");
```
Commented code, as we're already modifying

Reply:
I'll fix this up after the rebase